docs/cpp.md

```diff
@@ -17,66 +17,72 @@ You should then see the response `Hello faasm!`.
 
 # Writing functions
 
-Faasm aims to be uninvasive, allowing code to run natively _and_ in a serverless context. 
-To do this in C/ C++ functions we use a set of Faasm macros. These macros allow code to 
-be compiled with a standard toolchain and run natively, and with the Faasm toolchain, 
-and run in a serverless context.
+Faasm aims to be uninvasive, allowing code to run natively _and_ in a serverless
+context.  To do this in C/ C++ functions we use a set of Faasm macros. These
+macros allow code to be compiled with a standard toolchain and run natively, and
+with the Faasm toolchain, and run in a serverless context.
 
 The outline of this looks like:
 
 ```c++
 #include "faasm/faasm.h"
 
-FAASM_MAIN_FUNC() {
+int main(int argc, char* argv[]) {
     // Do something
 
     return 0;
 }
 ```
 
-Although for very simple functions, a standard `int main()` will also still work.
+Although for very simple functions, a standard `int main()` will also still
```

Comment:
Don't you think this is out of place without the macro?

Reply:
Ah yes good point, I'll modify in part 2